src/accelerate/commands/config/cluster.py

```diff
@@ -386,12 +386,21 @@ def get_cluster_input():
                 default=False,
                 error_message="Please enter yes or no.",
             )
-            fsdp_config["fsdp_sync_module_states"] = _ask_field(
-                "Do you want each individually wrapped FSDP unit to broadcast module parameters from rank 0 at the start? [YES/no]: ",
+            fsdp_config["fsdp_cpu_ram_efficient_loading"] = _ask_field(
+                "Do you want to enable CPU RAM efficient model loading? Only applicable for 🤗 Transformers. [YES/no]: ",
```

Comment:
Should we be a bit more clear? Like Transformer models? (I say this because the problem model in general was downloaded from `transformers`, so we should find a better way to specify this I think)

Reply:
Done